frontend/ui/network/manager.lua

```diff
@@ -1,6 +1,7 @@
 local ConfirmBox = require("ui/widget/confirmbox")
 local DataStorage = require("datastorage")
 local Device = require("device")
+local FFIUtil = require("ffi/util")
```

Comment:
and `local T = FFIUtil.template` at the bottom.

Reply:
this way?